runtime/web_worker.rs

```diff
@@ -528,7 +528,8 @@ impl WebWorker {
     };
 
     let bootstrap_fn_global = {
-      let context = js_runtime.global_context();
+      let realm = js_runtime.main_realm();
+      let context = realm.context();
```

Comment:
`JsRuntime::global_context` was renamed to `main_context`, so you don't need to go through the realm.